From e583892f5c5f2284d135624de27cfdfefcd53cd7 Mon Sep 17 00:00:00 2001
From: Leah Rowe <info@minifree.org>
Date: Sun, 17 Jul 2016 23:47:14 +0100
Subject: [PATCH] NOTFORMERGE: lenovo/t400: hard-code enable integrated-only
 video

Written with libreboot in mind. Libreboot uses native graphics
initialization only, so we want to ensure that these systems
only use the integrated (Intel) GPU for which native init exists.

Native graphics initialization does not yet exist for the ATI GPUs
on these laptops...

Change-Id: I2c056a8fb498972f87c4ec1122b239fdc9a4c666
Signed-off-by: Leah Rowe <info@minifree.org>
---
 src/drivers/lenovo/hybrid_graphics.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/drivers/lenovo/hybrid_graphics.c b/src/drivers/lenovo/hybrid_graphics.c
index 9b46646..2a6c300 100644
--- a/src/drivers/lenovo/hybrid_graphics.c
+++ b/src/drivers/lenovo/hybrid_graphics.c
@@ -64,7 +64,6 @@ static void hybrid_graphics_enable_peg(struct device *dev)
 	u8 hybrid_graphics_mode;
 
 	hybrid_graphics_mode = HYBRID_GRAPHICS_INTEGRATED;
-	get_option(&hybrid_graphics_mode, "hybrid_graphics_mode");
 
 	if (hybrid_graphics_mode == HYBRID_GRAPHICS_DISCRETE) {
 		/* connect LVDS interface to dGPU */
-- 
1.9.1

